vllm_ascend/platform.py

```diff
@@ -130,6 +130,27 @@ def check_and_update_config(cls, vllm_config: VllmConfig) -> None:
             enforce_eager = getattr(vllm_config.model_config, "enforce_eager",
                                     False)
 
+        additional_config = vllm_config.additional_config
+        parallel_config = vllm_config.parallel_config
+        cache_config = vllm_config.cache_config
+
+        if parallel_config:
+            # Default value for expert tensor parallel size
+            parallel_config.expert_tensor_parallel_size = 1
+
+            # NOTE: When enable_expert_parallel is True, we follow vLLM convention:
+            # ep_size = world_size, which means expert_tensor_parallel_size must be 1
+            if (additional_config
+                    and "expert_tensor_parallel_size" in additional_config
+                    and not parallel_config.enable_expert_parallel):
+                parallel_config.expert_tensor_parallel_size = int(
+                    additional_config["expert_tensor_parallel_size"])
+
+            # Calculate expert parallel size based on world size
+            parallel_config.expert_parallel_size = (
+                parallel_config.world_size //
+                parallel_config.expert_tensor_parallel_size)
```

Comment:
Sorry, I'm still confused. Could I understand like this: when disabling ep and enabling etp, a expected etp size is set, and an unexpected ep size is set?

or should ep size be set only when ep is enabled?
```suggestion
            if parallel_config.enable_expert_parallel:
                parallel_config.expert_parallel_size = (
                    parallel_config.world_size //
                    parallel_config.expert_tensor_parallel_size)
```